.github/workflows/job-postcommit-java-examples-dataflow.yml

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+# Runs Java Examples on Dataflow runner via cron and on push to master
+
+name: PostCommit Java Examples Dataflow
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+permissions: read-all
+
+jobs:
+  set-properties:
+    runs-on: [self-hosted, ubuntu-20.04]
+    outputs:
+      properties: ${{ steps.test-properties.outputs.properties }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          submodules: false
+      - id: test-properties
+        uses: ./.github/actions/setup-default-test-properties
+
+  postcommit-java-examples-dataflow:
+    needs: set-properties
+    name: Run PostCommit Java Examples Dataflow
+    runs-on: [self-hosted, ubuntu-20.04]
+    strategy:
+      fail-fast: false
+      matrix: #JDK11 execution will fail due to changes in ByteBuffer class since JDK9
+        #TODO: Retest once issue https://github.com/apache/beam/issues/23068 gets resolved.
+        #JDK17 execution will fail due to new access policies applied to the core JDK libraries starting at JDK 16.
+        #TODO: Retest once issue https://github.com/apache/beam/issues/23066 gets resolved.
+        java: ${{fromJson(needs.set-properties.outputs.properties).JavaTestProperties.SUPPORTED_VERSIONS}}
+        exclude: #Skipping Java 8 since there is no task for that version.
+          - java: "8"
```

Comment:
I'm a little confused. Why are we skipping Java 8, but talking about Java 11/9 in the comments?

We probably shouldn't be removing the Jenkins files here because of this exclusion (same thing as https://github.com/apache/beam/pull/23077#discussion_r988104603)

Reply:
Hi Danny!
About skipping Java 8: originally there was no Java 8 job for these tests. We have to manually skip it because Java 8 is included by default in the json file used to build the version matrix.
Versions 9 and 11 are being mentioned due to the changes made to JDK9 that affect JDK11 compilation.

I added back jenkins jobs since those tests don't execute successfully in GHAs yet.

Reply:
> About skipping Java 8: originally there was no Java 8 job for these tests. We have to manually skip it because Java 8 is included by default in the json file used to build the version matrix.

Skipping it is probably fine, but could we update the comment to be more informative (e.g. `Skipping for legacy reasons, we've never run Java 8 tests. These tests currently fail on Java 8.`). Even better, if including Java 8 happens to just work, we should just run the tests for that version.

Reply:
Other than that, this LGTM

Reply:
Hi Danny!
Turns out [execution with JDK8](https://github.com/fernando-wizeline/beam/actions/runs/3274968756/jobs/5389207261) worked like a charm.
Just had to add a new task to the build.gradle file and that did the trick.